### PR TITLE
PR: Algo bubble-sort in java

### PR DESCRIPTION
**Pull request by Sanchayan12**
This is PR for bubble sort in java

### CODE REVIEW

File: 1-data-structures/1-java/7-bubble sort
[the remark this answers is not in the document]
Comment (Sanchayan12) [reply 1-data-structures/1-java/7-bubble sort:1]: I have already change the name

**Comment by Abishethvarman** [comment]
Rename the title of the PR like "PR: Algo <algorithm name> in <programming language >"

**Comment by Sanchayan12** [comment]
I have made the changes

**Review by Thevapriyan07 — APPROVED** [review]

**Review by Abishethvarman — APPROVED** [review]
Approved

**Review by Abishethvarman — CHANGES_REQUESTED** [review]
"1-data-structures/1-java/7-bubble sort" this path should be "2-algorithm/1-java/7-bubble-sort.MD"

**Review by Abishethvarman — CHANGES_REQUESTED** [review]
You placed your file in data structure. Replace it in the Algorithm folder

**Comment by Abishethvarman** [comment]
Better close this PR and create a new

**Comment by Abishethvarman** [comment]
closed due to relocation issue

**Pull request closed without merging**